### PR TITLE
gRPC library for google iam mixins.

### PR DESCRIPTION
**Pull request by Landrito**

### CODE REVIEW

File: generated/python/grpc-google-iam-v1/PUBLISHING.rst [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+Note: This folder has been generated by the gen-api-package tool.
+
+The instructions assumes that no changes have been made to the folder and its
+contents since it was created.
+
+PREREQUISITES
+-------------
+
+- Python must installed
```
Comment (bjwatson) [inline generated/python/grpc-google-iam-v1/PUBLISHING.rst]: Typo. No need to regenerate, but can you do a quick fix in the packman template?

Comment (Landrito) [inline generated/python/grpc-google-iam-v1/PUBLISHING.rst]: By typo you mean missing period?

Comment (bjwatson) [inline generated/python/grpc-google-iam-v1/PUBLISHING.rst]: Bad grammar: "Python must *be* installed*.*"

Comment (Landrito) [inline generated/python/grpc-google-iam-v1/PUBLISHING.rst]: Ah Okay!

**Comment by bjwatson** [comment]
LGTM

**Pull request merged**